File: sa.cpp
#include<iostream>
using namespace std;
class P
{
	public:
P()
{
	cout<<"\n-parent con--\n";
}
};
class C:public P
{
public:
int a;
	C(int a){
this->a=a;
cout<<"\n---value of a is---- "<<a<<endl;
}
};
int main()
{
C obj(10);
return 0;
}

File: sa.java
import java.lang.*;
import java.io.*;
class P
{
	P()
	{
		System.out.println("\npar con\n");
	}
	void fun()
	{
		System.out.println("\n---concrete fun---\n");
	}
}
class C extends P
{
	int a
	
	C(int a)
	{
		this.a=a;		
	}
	void fun()
	{
		super();
		System.out.println("\n----concrete fun in child---\n");
	}	
}

class Main1()
{
:wq
:wq
	public static void main(String args[])
	{
		C obj=new C(10);
		obj.fun();
	}
}
